backend/src/api/user.rs

```diff
@@ -60,6 +60,13 @@ pub(crate) async fn register(
         roles: vec![],
         status: MembershipStatus::Pending,
         email: new.email,
+        phone: "".to_string(),
+        student_number: None,
+        nkbv_number: None,
+        sportcard_number: None,
+        ice_contact_name: "".to_string(),
+        ice_contact_email: "".to_string(),
+        ice_contact_phone: "".to_string(),
```

Comment:
Depends a bit on what we want to achieve here. Do we want to enforce that the marked data is provided on the first sign up? Probably yes.
If not, we should set the fields explicitly to ˋNoneˋ instead of an empty string (and adopt the DB fields accordingly).